.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - 9
```

Comment:
Doesn't flamingo run on node 12.14.1 ?

Reply:
I'm gonna remove this. AFAIK we don't use Travis for anything